docs/source/tutorials.rst

```diff
@@ -201,7 +269,7 @@ In order to create a new customized task, there are two places where you need to
 Set up a new environment
 +++++++++++++++++++++++++
 
-Environment classes are developed according to the `OpenAI Gym definition <https://www.gymlibrary.dev/content/environment_creation/>`__
+Environment classes are developed according to the `OpenAI Gym definition <https://gymnasium.farama.org>`__
```

Comment:
@raku-slyu - change to https://gymnasium.farama.org/api/env/

Reply:
Link updated in new commit